package.json

```diff
@@ -38,8 +38,8 @@
     "@edx/brand": "npm:@edx/brand-openedx@1.1.0",
     "@edx/frontend-component-footer": "10.2.1",
     "@edx/frontend-component-header": "^3.3.0",
-    "@edx/frontend-platform": "1.15.2",
-    "@edx/paragon": "19.25.3",
+    "@edx/frontend-platform": "^3.0.1",
```

Comment:
* [2.0.0](https://github.com/openedx/frontend-platform/releases/tag/v2.0.0): I'm concerned about some react-intl@v5 changes flying under the radar with unwanted effects.  Say, for instance [this change](https://unicode-org.github.io/icu/userguide/format_parse/messages/#quotingescaping) vs [this use](https://github.com/openedx/frontend-app-library-authoring/blob/master/src/library-authoring/author-library/messages.js#L206) of an escaped quotation mark.  Can you please check and confirm that we're fine with this and [other changes](https://github.com/openedx/frontend-platform/blob/master/docs/how_tos/i18n.rst#migrating-to-react-intl-5), and if not, to make the necessary modifications?  I don't think all of them are going to give us automated warnings.
 
* [3.0.0](https://github.com/openedx/frontend-platform/releases/tag/v3.0.0): Looks like we're not using caching anywhere - at least not knowingly - so we should be fine.